media/js/plugincheck/lib/plugincheck.js

```diff
@@ -14,17 +14,19 @@
      */
     function setJavaVersion() {
         var type;
-        var installedVersion;
+        var installedVersion = '0.0.0'; // fallback is better than returning undefined?
```

Comment:
Using a string like `unknown` would cause an error (like the one that broke the page here). It needs to be a version number in the form of a string. I figured this is better than a broken page.